src/interface.jl

```diff
@@ -130,17 +130,21 @@ is_directed(::Type{T}) where T = _NI("is_directed")
 
 Return true if `v` is a vertex of `g`.
 """
-has_vertex(x, v) = _NI("has_vertex")
+has_vertex(g, v) = _NI("has_vertex")
```

Comment:
Let's revert this. As discussed, `x` is used deliberately since it does not restrict itself to graphs. This function gets called on any two arguments.